server/utils/commonFunc.go

```diff
@@ -0,0 +1,59 @@
+package utils
+
+import (
+	"errors"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+func IsPassValid(password string) error {
```

Comment:
change the function signature to `IsPassValid(password string) (bool,error)`

Reply:
Done sir